### ISSUE TITLE
'-m'  in the fzf/faint pop-up window

### ISSUE THREAD

**Issue by har5ha** [salman-abedin/faint#4]
I get this strange '-m' in the fzf/faint pop-up window everytime.

https://i.imgur.com/vqYpRo6.png

TIA.

**Comment by salman-abedin** [comment]
Have you tried the latest version?

**Comment by har5ha** [comment]
Yes.. I Just did a git pull .. and it said "Already up to date"  then did a "sudo make install" . The above issue still exists on my system.

**Comment by salman-abedin** [comment]
Let me have a look at your `faint-operate` and `faint` files.

**Comment by har5ha** [comment]
Here you go .. 

[faint+faint-operate.zip](https://github.com/salman-abedin/faint/files/5536443/faint%2Bfaint-operate.zip)

**Comment by salman-abedin** [comment]
Everything looks fine there.  
I've tried to reproduce the issue but haven't been able to do so.  
I will see what I can do about this.  
In the mean time, try removing the `-m` flag from the `faint` file and see if it helps or not.
And also try running `find . -maxdepth 1` on your shell and see if you get any entry named `-m` on the standard output or not.

**Comment by har5ha** [comment]
Looks like the problem is with  --header ' '   [ ie. being empty somehow.. ]

I changed the line 27 to look like :
```

 25 FZF_DEFAULT_OPTS="\
 26    $FZF_DEFAULT_OPTS
 27    --reverse --border --no-info --margin 15%,30% --no-color -m --cycle --header '-----'
 28    --bind \"$KEY_LAUNCH:reload(faint-list -l {})+clear-query+top\"
 29    --bind \"$KEY_OPERATE:execute(faint-operate {+})+reload(faint-list -r)+clear-query+top\"
 30    --bind \"$KEY_BOOKMARK:execute(faint-bookmarks)+reload(faint-list -r)+clear-query+top\"
 31    --bind \"$KEY_BACK:reload(faint-list -b)+clear-query+top\"
 32    --bind \"$KEY_QUIT:cancel\"
 33    --bind \"$KEY_SHELL:execute(kill -35 $$)+abort\"
 34    --bind \"$KEY_EDIT:execute(faint-launch -e {})+clear-query+top\"
 35    --bind \"$KEY_MARK_ALL:toggle-all\"
 36    --bind \"$KEY_MARK_DOWN:toggle-down\"
 37    --bind \"$KEY_MARK_UP:toggle-up\"
 38    --bind \"$KEY_UP:up\"
 39    --bind \"$KEY_DOWN:down\"
 40    --bind \"$KEY_REFRESH:reload(faint-list -r)\"
 41    "
 42 export FZF_DEFAULT_OPTS
```

Now I get this 
https://imgur.com/GaeH2fL.png
which is kinda acceptable instead of an "empty" string being the header (which seems to be causing the issue)

PS : 
I can't seem to put anything else (ie. space etc.) other than a visible character  to make it to work with the --header option. So I chose '------' as a separator/header . Whether it is an Escaping of Character(s)/String(s) issue, I don't know.

**Comment by har5ha** [comment]
> And also try running find . -maxdepth 1 on your shell and see if you get any entry named -m on the standard output or not.

I didn't get any entry name -m in the output.

**Comment by har5ha** [comment]
If I run fzf with the same "FZF_DEFAULT_OPTS" in faint (ie. --header '' ) in the terminal. 

`$ /usr/bin/fzf --reverse --border --no-info --margin 15%,30% --no-color -m --cycle --header '' `

It shows up as an empty line as header in the fzf-popup window, like it is supposed to.

**Comment by salman-abedin** [comment]
I see. In that case, replace the single quotes with double quotes and escape them using backslashes.  
It's gonna look like this `--header \"\"`.  
I'm gonna push with this on the next commit so that you don't have to worry about compatibility with the new versions.

**Comment by har5ha** [comment]
`--header \"\"`

I tried that option earlier too.. But then I was getting this error, from the very next line in "faint" file:
```
$faint
unknown option: ;:reload(faint-list -l {})+clear-query+top
```

```
 27    --reverse --border --no-info --margin 15%,30% --no-color -m --cycle --header \"\"
 28    --bind \"$KEY_LAUNCH:reload(faint-list -l {})+clear-query+top\"
```

**Comment by salman-abedin** [comment]
Okay then try removing the `export FZF_DEFAULT_OPTS` line altogether and put the `export` command before the `FZF_DEFAULT_OPTS` variable. It's gonna look like this...  
```sh
export FZF_DEFAULT_OPTS="\
   $FZF_DEFAULT_OPTS
   --reverse --border --no-info --margin 15%,30% --no-color --header ''
...
...
...
```

**Comment by har5ha** [comment]
> Okay then try removing the `export FZF_DEFAULT_OPTS` line altogether and put the `export` command before the `FZF_DEFAULT_OPTS` variable. It's gonna look like this...
> 
> ```shell
> export FZF_DEFAULT_OPTS="\
>    $FZF_DEFAULT_OPTS
>    --reverse --border --no-info --margin 15%,30% --no-color --header ''
> ...
> ...
> ...
> ```
I am getting the same error as before. 

```
$faint
unknown option: ;:reload(faint-list -l {})+clear-query+top
```

```
 $ fzf --version 
0.20.0
```

**Comment by salman-abedin** [comment]
Let me have a look at your `faint-config` file

**Comment by har5ha** [comment]
Here you go ..
[faint-config.zip](https://github.com/salman-abedin/faint/files/5540195/faint-config.zip)

**Comment by salman-abedin** [comment]
I think your `back` and `launch` keys are causing the issue.  
Try a few things like double quoting, no quoting and escaping to see what works for you.  
Fingers crossed.  
P.S.: Try pulling down the new version to see if it helps or not.

**Comment by har5ha** [comment]
Thanks for all your help. I am happy with making the header '---' to avoid the issue, for now. May be its an issue with my version of fzf or my setup.

**Comment by salman-abedin** [comment]
You better not give up on me man! lol.  
We'll figure this out. 🙂

**Comment by har5ha** [comment]
Definitely not!. Sure thing..
